stream-handler/handlers/apply_user_filter_handler.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
Could you add a comment briefly explaining what the ApplyUserFilterHandler does and that it is intended to be called after RecordUserHandler?